mission/battery_monitor/launch/battery_monitor.launch

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0"?>
-<launch>
-    <node pkg="battery_monitor"
-    type="battery_monitor.py"
-    name="battery_monitor"
-    output="screen"/>
-</launch>
```

Comment:
The old launch file is deleted, but there is no new one created. A launch file for internal status should be created and called from gladlaks.launch

Reply:
A new one was created, though it was called internal_status.launch for clarity's sake. Though it was not called for in gladlaks.launch